README.md

```diff
@@ -2,33 +2,29 @@ Terraform Provider for Google Cloud Platform
 ==================
 
 - Website: https://www.terraform.io
```

Comment:
This is the website for terraform, but this provider isn't on it. Instead of doing this here, I actually wonder if you could do an in-text link in the initial description like "The google provider is a plugin that allows [Terraform](https://www.terraform.io) to manage infrastructure on Google Cloud Platform."